django/db/models/base.py

```diff
@@ -692,7 +692,7 @@ def refresh_from_db(self, using=None, fields=None, from_queryset=None):
             self._prefetched_objects_cache = {}
         else:
             prefetched_objects_cache = getattr(self, "_prefetched_objects_cache", ())
-            fields = list(fields)
+            fields = set(fields)
             for field in list(fields):
```

Comment:
Just to make it clearer, maybe also a tiny bit faster
```suggestion
            for field in fields.copy():
```

Reply:
Right ✅